<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>themnguoi</class>
 <widget class="QDialog" name="themnguoi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_phone">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Số điện thoại:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_huy_tn_23">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 240, 223);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hủy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sdt_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_fullname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_email_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_hoten_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_diachi_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_email">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_diachi">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>290</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Địa chỉ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_title">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 22pt &quot;Times New Roman&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Hồ sơ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_luu_tn_23">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(121, 240, 223);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_user_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mật khẩu</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mk_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>370</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhSensitiveData</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_email_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mkl_tn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_address_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>410</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nhập lại mật khẩu </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
